lib/r10k/environment/base.rb

```diff
@@ -44,14 +44,14 @@ def initialize(name, basedir, dirname, options = {})
     @dirname = dirname
     @options = options
     @puppetfile_name = options.delete(:puppetfile_name)
-    @overrides = options.delete(:overrides)
+    @overrides = options.delete(:overrides) || {}
 
     @full_path = File.join(@basedir, @dirname)
     @path = Pathname.new(File.join(@basedir, @dirname))
 
     @puppetfile  = R10K::Puppetfile.new(@full_path,
-                                        {overrides: @options[:overrides],
```

Comment:
This was a bug that was uncaught earlier, but exposed with the additional testing that comes from attempting to actually use this data at the Module level. It accidentally got rolled into a refactor commit but I'm happy to break it out if that would be better.